core/controllers/profile.py

```diff
@@ -69,8 +69,8 @@ def get(self, username: str) -> None:
 
         subscriber_ids = subscription_services.get_all_subscribers_of_creator(
             user_settings.user_id)
-        is_already_subscribed = (self.user_id in subscriber_ids)
-        is_user_visiting_own_profile = (self.user_id == user_settings.user_id)
+        is_already_subscribed = self.user_id in subscriber_ids
```

Comment:
Out of interest, are the changes in this file forced by the linter?

I actually think the old presentation is clearer (and less error prone, especially in the case of A = B == C) but my opinion here isn't super-strong.

Reply:
Yes, the linter now enforces the removal of unnecessary parentheses